wd_helper.au3

```diff
@@ -687,43 +676,52 @@ EndFunc   ;==>_WD_HighlightElement
 ;                  2 - Highlight yellow rounded box
 ;                  3 - Highlight yellow rounded box + border  dotted red
 ; Return values .: Success - True
-;                  Failure - False and sets @error returned from _WD_ExecuteScript()
+;                  Failure - False and sets @error to _WD_ERROR_InvalidArgue or the error code from _WD_ExecuteScript()
 ; Author ........: Danyfirex
-; Modified ......: mLipok
+; Modified ......: mLipok, Danp2
 ; Remarks .......:
 ; Related .......: _WD_HighlightElement
 ; Link ..........: https://www.autoitscript.com/forum/topic/192730-webdriver-udf-help-support/?do=findComment&comment=1396643
 ; Example .......: No
 ; ===============================================================================================================================
-Func _WD_HighlightElements($sSession, $aElements, $iMethod = Default)
+Func _WD_HighlightElements($sSession, $vElements, $iMethod = Default)
 	Local Const $sFuncName = "_WD_HighlightElements"
 	Local Const $aMethod[] = _
 			[ _
-			"border: 0px", _
-			"border: 2px dotted red", _
+			"border: 0px;", _
+			"border: 2px dotted red;", _
 			"background: #FFFF66; border-radius: 5px; padding-left: 3px;", _
 			"border: 2px dotted red; background: #FFFF66; border-radius: 5px; padding-left: 3px;" _
 			]
-
-	Local $iHighlightedElements = 0
+	Local $sScript, $sResult, $iErr, $sElements
 
 	If $iMethod = Default Then $iMethod = 1
 	If $iMethod < 0 Or $iMethod > 3 Then $iMethod = 1
 
-	For $i = 0 To UBound($aElements) - 1
-		$aElements[$i] = __WD_JsonElement($aElements[$i])
-	Next
+	If IsString($vElements) Then
+		ConsoleWrite('@@ Debug(' & @ScriptLineNumber & ') : $vElements = ' & $vElements & @CRLF & '>Error code: ' & @error & @CRLF) ;### Debug Console
```

Comment:
ConsoleWrite ?
Why not   __WD_ConsoleWrite  ?

Reply:
That shouldn't even be there. It's left over from my testing. 🙄